app/forms/openid_connect_authorize_form.rb

```diff
@@ -15,6 +15,7 @@ class OpenidConnectAuthorizeForm
     redirect_uri
     response_type
     state
+    biometric_comparison_required
```

Comment:
Couple questions:

- Should we be checking this against an allowlist of SPs, or considering the feature flag?
- To the prior comment about `'true'` (string) vs. `true`, should we do that normalization as early as possible, in this form class

Reply:
> Should we be checking this against an allowlist of SPs, or considering the feature flag?

I think we should render an error eventually, but that will be part of the stories that will go under [LG-11599](https://cm-jira.usa.gov/browse/LG-11599) (the epic for setting up the OIDC and SAML interface)

> To the prior comment about 'true' (string) vs. true, should we do that normalization as early as possible, in this form class

Hmm, let me see how that looks.

Reply:
> To the prior comment about 'true' (string) vs. true, should we do that normalization as early as possible, in this form class

This has been addressed